api/models/prediction_job.go

```diff
@@ -29,11 +29,11 @@ type PredictionJob struct {
 	// Keep metadata internal until it's needed to be exposed
 	Metadata Metadata `json:"-"`
 	// The field name has to be prefixed with the related struct name
-	// in order for gorm Preload to work with association_foreignkey
+	// in order for gorm Preload to work with references
 	VersionID       ID           `json:"version_id"`
 	VersionModelID  ID           `json:"model_id"`
 	ProjectID       ID           `json:"project_id"`
-	Environment     *Environment `json:"environment" gorm:"association_foreignkey:Name;association_autoupdate:false;association_autocreate:false;"`
```

Comment:
No other tables are using the `association_autoupdate:false;association_autocreate:false` tags. As such, it only affects the `Environment` and should be safe to remove.